### PR TITLE
add content to MIDI pattern sequencer

### PR DESCRIPTION
**Pull request by Aldwin-b**

### CODE REVIEW

File: docs/reference-manual/processes/library/patternist.md [hunk 1]
```diff
@@ -9,3 +9,24 @@ grand_parent: Reference
 
 permalink: /processes/patternist.html
 ---
+# MIDI Pattern Squencer
+
+![Patternist]({{ site.img }}/reference/processes/patternist.png "MIDI Squencer Pattern Example")
+
+Score has a MIDI pattern sequencer that you can use.
+On the left of the squencer, there is te actual note of the lane with a corresponding number.
```
Comment (jcelerier) [inline docs/reference-manual/processes/library/patternist.md:17]: te -> the

File: docs/reference-manual/processes/library/patternist.md [hunk 2]
```diff
@@ -9,3 +9,24 @@ grand_parent: Reference
...
+Each steps correspond to a measure. If the step is activate ( _yellow_), the note will be played.
```
Comment (jcelerier) [inline docs/reference-manual/processes/library/patternist.md:18]: Each steps -> each step
activate -> activated

**Review by jcelerier — CHANGES_REQUESTED** [review]

**Review by Aldwin-b — COMMENTED** [review]
"te" replaced by "the"
"steps" by "step"
and "activate" by "activated"

**Comment by Aldwin-b** [comment]
Image added

**Pull request merged**